config/default/migrate_plus.migration.hs_capx.yml

```diff
@@ -108,8 +108,8 @@ source:
       selector: education
     -
       name: department
-      label: Departement
-      selector: academicOffices/0/department
+      label: Department
+      selector: academicOffices
```

Comment:
What happens if you do `academicOffices/*/department` without changing the process methods?

Reply:
Definitely will try that. I think I got used to the simpleXML selectors, and this is a JSON import. I didn't know about the asterisk/wildcard (is there a list of these somewhere?).

Reply:
@pookmish I tried using `academicOffices/*/department` as the selector with the original process methods, but didn't have any luck.

Reply:
Yea it doesn't look like the default JSON `data_parser_plugin` for `migrate_plus` includes any special selectors. There is an open issue to add a JSONPath parser plugin to allow for that: https://www.drupal.org/project/migrate_plus/issues/3007709

Reply:
oof, 5 years old. ouch. Well that was worth a shot. guess we'll have to do the subprocess approach.

Reply:
Yea I'm a little surprised the JSONPath support hasn't got more traction.

Reply:
I was able to re-implement the `null_coalesce`.